docs/changelogs/latest.rst

```diff
@@ -56,6 +57,26 @@ OS compatibility is subject to the following limitations:
 Breaking Changes
 ----------------
 
+IPTables Rule Refinement
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+**IPTables does NOT have breaking changes out of the box**
```

Comment:
IMHO this should be in an Important box or the likes so it stands out.  The bold text does not stand out when rendered.